api.go

```diff
@@ -373,15 +373,30 @@ func (api *API) delSilence(w http.ResponseWriter, r *http.Request) {
 }
 
 func (api *API) listSilences(w http.ResponseWriter, r *http.Request) {
-	sils, err := api.silences.All()
+	var (
+		num    = r.FormValue("n")
```

Comment:
nitpick, but why is one parameter in the long form and the other short? I'd vote for consistency. Maybe use `page` instead?